search-app/api_docs.md

```diff
@@ -1654,19 +1654,19 @@ Find granules with cloud cover numeric range set to min: -70.0 max: 120.0
 
 #### <a name="g-platform"></a> Find granules by platform
 
-This supports `pattern`, `ignore_case` and option `and`.
+This supports `pattern`, `ignore_case`, `exclude_collection` and option `and`.
```

Comment:
I think we need to explain what this option is doing, especially since it's only effective for the granules without any platforms and not having any impact on other granules.